### DISCUSSION TITLE
iNotify library missing

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2197] [Ideas] [answered]
While its been stated that dependencies won't be added, I've observed several of them being added via a bug report.
-e NEXTCLOUD_ADDITIONAL_PHP_EXTENSIONS="imagick inotify" ^

Please add inotify by default.

**Comment by szaimen** [comment, marked as answer]
Hi, we have no intention to add this by default since this is not a core feature from our point of view. But you can already add it by yourself if you need it using this exact feature.
